ionic.config.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
@charlesoma you added this config. What does it do in relation to the button size?

Reply:
Must have been in error. Taking it out